src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseGradingService.java

```diff
@@ -571,97 +599,189 @@ private boolean createFeedbackForDuplicateTests(Result result, ProgrammingExerci
             String duplicateDetailText = "This is a duplicate test case. Please review all your test cases and verify that your test cases have unique names!";
             List<Feedback> feedbacksForDuplicateTestCases = duplicateFeedbackNames.stream()
                     .map(feedbackName -> new Feedback().type(FeedbackType.AUTOMATIC).text(feedbackName + " - Duplicate Test Case!").detailText(duplicateDetailText).positive(false))
-                    .collect(Collectors.toList());
+                    .toList();
             result.addFeedbacks(feedbacksForDuplicateTestCases);
+
             // Enables to view the result details in case all test cases are positive
             result.setHasFeedback(true);
             String notificationText = TEST_CASES_DUPLICATE_NOTIFICATION + String.join(", ", duplicateFeedbackNames);
             groupNotificationService.notifyEditorAndInstructorGroupAboutDuplicateTestCasesForExercise(programmingExercise, notificationText);
+
             return true;
         }
-        return false;
+        else {
+            return false;
```

Comment:
I suggest reverting this change. In the current notation, `false` gets used as a fallback for the method. In your notation, it gets used as a fallback for the if clause. In this instance, it's the same, but as far as I know, we predominantly use the former notation, the method fallback, in our code.

Reply:
reverted